### DISCUSSION TITLE
When I turned off my Wi-Fi and turned on back, GitHub widget gives that error. I already gave Background App Refresh permission but it can’t able to fetch data until I manually fix. (iOS 15.2)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#9344] [Mobile]
![image](https://user-images.githubusercontent.com/95879997/147506121-31c48adf-8b80-45a2-b8ff-a2e6cbf3707d.jpeg)

**Comment by brianlovin** [comment]
Thanks for the heads up! We have an open issue to improve the fallback for this widget, I'll add your image there.
